docs/thechickeneater.md

```diff
@@ -0,0 +1,32 @@
+Notes on the github.com user [thechickeneater](https://github.com/thechickeneater)
+---
+Entitlements requires access in order to do things like:
+- modify LDAP groups and memberships
+- modify github.com team memberships
+- manage stafftools access
+
+Originally, @kpaulisse created `thechickeneater` as a service account that could perform Entitlements' many tasks. He
+did this at @sigusrbacon's request to avoid making the account a target of attack. However, the obscurity of the 
+account caused occasional mini-fire drills when Hubbers would discover some unknown account `thechickeneater` as a 
+hard-coded user in the monolith, and elsewhere (like an org admin). To avoid that, it was decided to create a more 
+obvious `entitlements` user.
+
+> [!NOTE]  
+> "sigusrbacon told me to come up with something non-obvious but bears no responsibility for my love of chicken
+> sandwiches becoming a hard-coded part of the most sensitive GitHub account. I didn’t want this to have to be a
+> GitHub account at all. I just wanted an anonymous API token or something. But the staff tools team wasn’t willing
+> or able to do that. And we specifically didn’t want to use hubot because it was too powerful already" --kpaulisse 09 May 2024
+
+By the time we switched over to the `entitlements` user, `thechickeneater` was already [hard-coded in stafftools]
+(https://github.com/search?q=repo%3Agithub%2Fgithub+entitlements_master&type=code), already granted the
+ `entitlements-master` role, and there was no longer any funding to make stafftools changes, and no stafftools team.
+
+To this day, `thechickeneater` still makes promotions/demotions to `site_admin` (ie grants stafftools `access` role) 
+and manages Hubbers' `stafftools_role`s (ie developer, super-admin). In order to change this, you must understand that 
```

Comment:
I think you mean `e.g.` here and above. But either way, needs punctuation.

https://www.merriam-webster.com/grammar/ie-vs-eg-abbreviation-meaning-usage-difference

```suggestion
and manages Hubbers' `stafftools_role`s (e.g., developer, super-admin). In order to change this, you must understand that 
```